### PR TITLE
InstructorFeedbackResults, RQG/RGQ view: no response texts are not italicized #3181

### LINKED ISSUE #3181: InstructorFeedbackResults, RQG/RGQ view: no response texts are not italicized
As seen below, the text for "There are no responses ..." are italicized:
(GRQ/GQR view)

(Question view)

However the convention doesn't seem to be followed in RQG/RGQ view...

### PR DESCRIPTION
**Pull request by lejolly**
Fixes #3181

### CODE REVIEW

File: src/main/webapp/jsp/instructorFeedbackResultsByRecipientGiverQuestion.jsp [hunk 1]
```diff
@@ -251,7 +251,7 @@
                                     <span class='glyphicon glyphicon-chevron-up pull-right'></span>
                                 </div>
                                 <div class='panel-collapse collapse in'>
-                                    <div class="panel-body"> There are no responses received by this user 
+                                    <div class="panel-body"> <i>There are no responses given by this user</i>
```
Comment (kanghj) [inline src/main/webapp/jsp/instructorFeedbackResultsByRecipientGiverQuestion.jsp]: Why change from "received" to "given"? The panels are displayed by recipient on this page.

**Comment by kanghj** [comment]
Also you should not prefix your commit messages with the issue number in square brackets. We save that format for merge commits to master

**Comment by lejolly** [comment]
Noted, have fixed the words.

**Comment by kanghj** [comment]
I am not able to pass InstructorFeedbackResultsPageUiTest on line 268 <code>resultsPage.verifyHtmlAjax("/instructorFeedbackResultsAjaxByRQG.html");</code>.

You need to run the tests and ensure that nothing to failing because of your changes.

**Pull request merged**